cmd/yurt-controller/OWNERS

```diff
@@ -0,0 +1,9 @@
+# See: https://go.k8s.io/owners
```

Comment:
i think we need to add OWNERS for every controllers, not need to add OWNERS for yurt-controller-manager component.

Reply:
This is just an owner restriction on the frame entry. Adding an owner to each component is not affected